contracts/core/GlobalReentrancyLock.sol

```diff
@@ -68,46 +68,36 @@ contract GlobalReentrancyLock is IGlobalReentrancyLock, CoreRefV2 {
 
     /// @notice cache the address that locked the system
     /// only this address can unlock it
-    address private _sender;
+    address public sender;
 
     /// @notice store the last block entered
     /// if last block entered was in the past and status
     /// is entered, the system is in an invalid state
     /// which means that actions should be allowed
-    uint88 private _lastBlockEntered;
+    uint88 public lastBlockEntered;
 
     /// @notice system lock level
-    uint8 private _lockLevel;
+    uint8 public lockLevel;
 
     /// @param core reference to core
     constructor(address core) CoreRefV2(core) {}
 
     /// ---------- View Only APIs ----------
 
-    /// @notice view only function to return the last block entered
-    function lastBlockEntered() external view returns (uint88) {
-        return _lastBlockEntered;
-    }
-
     /// @notice returns the last address that locked this contract
     function lastSender() external view returns (address) {
-        return _sender;
+        return sender;
     }
```

Comment:
you could directly rename the state variable `lastSender`

Reply:
done